config/secrets.defaults.yml

```diff
@@ -0,0 +1,8 @@
+---
+test:
+  kubevirt_defaults: &kubevirt_defaults
+    hostname: KUBEVIRT_HOSTNAME
+    token: KUBEVIRT_TOKEN
+    metrics_hostname: METRICS_HOSTNAME
+  kubevirt:
+    <<: *kubevirt_defaults
```

Comment:
I can't remember if we changed this yet or not.  cc @jrafanie

Reply:
Not yet.  I haven't had time to get back to it.  The referenced changes are coming from this core change: https://github.com/ManageIQ/manageiq/pull/23292 and how each plugin would use these changes: https://github.com/ManageIQ/manageiq-providers-vmware/pull/928

Reply:
I might just leave the warnings with rails 7.1 for now and people can fix them when their usage of rails secrets in 7.1 doesn't work or gets a warning.  Note, I don't think we actually use rails to store secrets encrypted.  That's the part that's completely gone in rails 7.1.  Access is probably gone in 7.2.